front/src/reducers/osrdconf/stdcmConf/stdcmConfReducers.spec.ts

```diff
@@ -1,15 +1,17 @@
+import { infraState } from 'reducers/infra';
 import { OsrdStdcmConfState, StandardAllowance } from 'applications/operationalStudies/consts';
 import { createStoreWithoutMiddleware } from 'Store';
 
 import { describe, expect } from 'vitest';
 import { stdcmConfInitialState, stdcmConfSlice, stdcmConfSliceActions } from '.';
-import testCommonConfReducers from '../common/tests/utils';
+import testCommonConfReducers from '../osrdConfCommon/__tests__/utils';
 
 const createStore = (initialStateExtra?: Partial<OsrdStdcmConfState>) =>
   createStoreWithoutMiddleware({
     [stdcmConfSlice.name]: {
       ...stdcmConfInitialState,
       ...initialStateExtra,
+      ...infraState,
```

Comment:
Done

Reply:
InfraState is the initial state of infra (infraId and featureInfoClick)

Reply:
I agree with Valentin, it's ok if some params are missing since they wiil filled by the rootInitialState (see in Store.ts, createStoreWithoutMiddleware)